src/components/PersonRoleDisplay/PersonRoleDisplay.tsx

```diff
@@ -5,7 +5,7 @@ type Props = {
   personRole?: PersonRole | undefined | null
 }
 
-const PersonRoleDisplay: React.FC<Props> = ({ personRole }) => (
+const PersonRoleDisplay: React.FC<Props> = ({ personRole }: Props) => (
```

Comment:
I don't get this one? `:Props` can be removed because this type is already declared here `<Props>`